corehq/apps/hqadmin/management/commands/record_deploy_success.py

```diff
@@ -133,6 +134,14 @@ def handle(self, **options):
                 environment=options['environment'], user=options['user'],
                 compare_url=compare_url)
             call_command('mail_admins', message_body, **{'subject': 'Deploy successful', 'html': True})
+            try:
+                recipient = settings.DAILY_DEPLOY_EMAIL
+                subject = '[%] Deploy Successful'.format(options['environment'])
+                send_HTML_email(subject=subject,
+                                recipient=recipient,
+                                html_content=message_body)
+            except Exception:
```

Comment:
This seems like a very large hammer. I would have expected something like `except AttributeError` to be the only permissible error here (if the setting is not configured)